hnn_core/network.py

```diff
@@ -161,6 +161,8 @@ class Network(object):
         'common', or any of the elements of the cellname or feedname lists.
     cell_response : CellResponse
         An instance of the CellResponse object.
+    feed_times : dict of list (n_trials) of list (n_cells) of list (n_times)
```

Comment:
Is this true for all feed types? I can see how e.g. `feed_times['evprox_1']` is a list of list but it almost looks like `feed_times['tonic']` is a dict of dict?

Reply:
I agree it's not super-clear. Which is why I already have a proposal ready in #221 ;) Tonic input is clearly a 'bias', which can have different semantics from the 'drivers'.

Reply:
I tried to clarify it a bit but I leave it to @cjayb to improve it further in #221